velox/docs/functions/presto/datetime.rst

```diff
@@ -94,7 +94,12 @@ Date and Time Functions
 
 .. function:: from_iso8601_date(string) -> date
 
-    Parses the ISO 8601 formatted ``string`` into a ``date``.
+    Parses the ISO 8601 formatted date ``string`` into a ``date``.
+    ISO 8601 date ``string`` can be formatted as any of the following:
+    ``[+-]YYYY``
```

Comment:
Do we allow number of digits in the year to be less or greater than 4? If so, would be nice to clarify. It would be great to add a few examples.

Reply:
Yes - the year must simply be at least one digit. In theory it can go up to 6-7 digits before it begins to overflow, and then errors out at 8 digits, though I suppose such values are out of the realm of real-world usage. 

Updated the format syntax in the docs to better reflect this, and added a note about year values needing to be between 1 and 6 digits:

```
    [+-][Y]Y*
    [+-][Y]Y*-[M]M
    [+-][Y]Y*-[M]M*-[D]D
    [+-][Y]Y*-[M]M*-[D]D *
```